TODO.rst

```diff
@@ -37,21 +37,24 @@ Calibration
 
 Frequency 
 -----------
-* add write() method 
-
+* add write() method
 
 vi 
 --------------------
 * re-write VNA's coherently 
 * use consistent method names so VNA's are inter-changable as much as possible
 * make a generic VNA that uses IDN? to load the correct class (durrants idea)
 
+VectorFitting
+-------------
+* implement passivity evaluation and enforcement routines
```

Comment:
You remind me that TODO file exists, thanks ;)
It is probably obsolete in few things.